cli/command/container/stats.go

```diff
@@ -123,6 +125,8 @@ func RunStats(ctx context.Context, dockerCLI command.Cli, options *StatsOptions)
 
 	showAll := len(options.Containers) == 0
 	if showAll {
+		span := trace.SpanFromContext(ctx)
```

Comment:
When in doubt, [create a nested span](https://opentelemetry.io/docs/instrumentation/go/manual/#create-nested-spans). It provides more detailed tracing insights than reusing a single top-level span for the entire lifetime of the process.